docs/faq.rst

```diff
@@ -4,23 +4,94 @@ Frequently Asked Questions
 How to ...
 -----------
 
-... Run examples?
+**... Run examples?**
     PARETO project provides examples to run the operational produced water management model
     and the strategic produced water management model (see pareto/case_studies/).
     To run the examples, go to:
 
     * pareto/operatinal_water_management/run_operational_model.py
     * pareto/strategic_water_management/run_strategic_model.py
 
-... Get more help?
+
+**... Get more help?**
     Use the website to `register <https://pareto.org/register/>`_ for the PARETO support mailing list.
     Then you can send questions to |support-email|. For more specific technical questions, we recommend
     our new `PARETO discussions board on Github <https://github.com/project-pareto/discussions>`_.
 
 Troubleshooting
 ---------------
 
-Missing win32api DLL
+**No available solver found among choices: (('cbc',),)**
+
+  If you encounter the error message "No available solver found among choices: (('cbc',),)," 
+  it typically means that the solver 'cbc' is not installed or not correctly configured in 
+  your environment. Here are steps to resolve this issue:
+
+  1. **Solver Installation**
+
+    Ensure that the 'cbc' solver is installed on your system. You can install it using the following methods:
+
+    * **For Windows Users:**
+      The easiest way to get Cbc on Windows is to download an archive from https://github.com/coin-or/Cbc/releases.
+
+    * **For Mac OS users:**
+
+      .. code-block:: bash
+
+          $ brew tap coin-or-tools/coinor
+          $ brew install coin-or-tools/coinor/cbc
+
+    * **For Ubuntu users:**
+
+      .. code-block:: bash
+
+          $ sudo apt-get install coinor-cbc
+
+  2. **Verify Installation**     
+
+    After installation, verify that the solver is accessible from your command line::
+
+        cbc
+
+    You should see the cbc solver interface if it is correctly installed.
```

Comment:
When I tried this step in my conda environment. I got the following error message:

    'cbc' is not recognized as an internal or external command,
operable program or batch file.

 I think this might be because I installed CBC using the `idaes get-extensions` command. I think perhaps this whole step can just be eliminated - the subsequent step serves essentially the same purpose, and could be described as "Verify Installation and Configure Solver in Your Code".